serverless/pytorch/facebookresearch/sam/nuclio/function-gpu.yaml

```diff
@@ -0,0 +1,66 @@
+metadata:
+  name: pth.facebookresearch.sam.vit_h
+  namespace: cvat
+  annotations:
+    name: Segment Anything
+    version: 2
+    type: interactor
+    spec:
+    framework: pytorch
+    min_pos_points: 1
+    min_neg_points: 0
+    help_message: The interactor allows to get a mask of an object using at least one positive, and any negative points inside it
```

Comment:
Need to add gif from hrnet. Thus UI will show visual help how to work with the model.